src/Parser/DisciplinePriest/Modules/Features/AtonementAttribution.js

```diff
@@ -0,0 +1,30 @@
+import Module from 'Parser/Core/Module';
+import isAtonement from '../Core/isAtonement';
+
+class AtonementAttribution extends Module {
+
+  reorderEvents(events) {
+    // Push down atonement healing on self to the next atonement healing
```

Comment:
Why?

Reply:
The idea behind this is, right now the way atonement attribution is made is by simply associating a damage event to the next atonement events until it hits another damaging event. With the latency on self heal, if 2 damaging events occur simultaneously, the healing event of the first damaging event will heal the player before the 2nd damage event hits. Pushing it down groups it with the associated event. We can then handle the double damaging event issue

Reply:
Could you put that in a comment in the method please?